### DISCUSSION TITLE
What is the future roadmap for Buffalo?

### DISCUSSION THREAD

**Discussion by preslavrachev** [gobuffalo/buffalo#2204] [General]
I was wondering if anyone could share a roadmap for the future of Buffalo. Is the framework still in active development?

**Comment by paganotoni** [comment]
Hi @preslavrachev! This is a fair and great question. 

From what I know about the Buffalo team there are a few initiatives going on right now. These are mostly related with the CLI. The most recent one is related with the transition from Packr into the stdlib `embed` package.

There is a "Road to Buffalo 1.0" document that may state some of the priorities that the core contributors have. So in summary there is no high level roadmap beyond that document. 

No in terms of the framework being actively in development I would say it's not like 2 years ago, however I know at least 6 folks who are still putting time and effort many buffalo libraries and tools.

**Comment by preslavrachev** [comment]
Well, I honestly want the development to continue. I also checked that Road to 1.0 document, and it has been 4 since it was published, sadly. 

I know what most people in the Go community think about Rails-like frameworks that do things for you, but I don't share that sentiment. I am a pragmatic person who needs to fulfill project requirements at the end of the day. If every time I started a new project, I had to write HTTP handling, DB access, event management, app bundling, authentication and all the rest by hand, I'd need one spare life.

I guess, what I am trying to say is that I am testing the waters to see if the Go community is ready for the return of a mature MVC Web framework. I've been chatting with @brendensoares over at the [Revel](https://github.com/revel/revel/) repo, and he seems to kind of see things the same way. 

The question is whether the problem needs a technical solution first, or is it more about educating the community about the demand. Then, we can sit down and identify the direction forward.

**Comment by paganotoni** [comment]
@preslavrachev thanks for being patient on this. @sio4 @fasmat and I have been talking about this. We're working on a new "Road to 1.0" document that will post in the repo.

**Comment by preslavrachev** [comment]
@paganotoni @sio4 @fasmat Congrats on the great work around the latest release! This gives me hope that we regain much of the early critical mass, and I am eager to help however I can. Besides contributing in pure code, I happen to write a lot. I maintain a newsletter over at https://goingwithgo.com/, in which I am trying to share adoption stories from around the Go community. Perhaps we could share a brief "interview" with some of you there? Does not have to be formal in any way - just a few words about who you are, how you jumped into Go, why Buffalo matters, etc.

If you are interested, you can always write to me at "hey at preslav.me".

Cheers!
